src/BTCPayServer.Lightning.All/ConnectChannels.cs

```diff
@@ -50,8 +50,9 @@ public static async Task ConnectAll(RPCClient cashCow, IEnumerable<ILightningCli
         }
         public static async Task CreateChannel(RPCClient cashCow, ILightningClient sender, ILightningClient dest)
         {
+            var amount = new LightMoney(1234); // use arbitrary amount to check if channel exists
```

Comment:
Using this arbitrary amount to distinguish it from the value used in `CanPayInvoiceAndReceive`.